### PR TITLE
fixed some bugs

### PR DESCRIPTION
**Pull request by IsQiao**
1.fixed Candlestick 1 day  cycle bug
2.fixed actionjob debug mode throw exception

### CODE REVIEW

File: src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js [hunk 1]
```diff
@@ -1,2 +1,96 @@
...
+            // 给chartConfig添加展示周期
+            chartConfig.interval =defaultActiveCycle;
+            // 给chartConfig添加展示产品
+            chartConfig.symbol = `${window.tokenId}/EOS`// index_market
+
+            // 初始化 TradingView
+            widget = new window.TradingView.widget(chartConfig)
+
+            widget && widget.onChartReady && widget.onChartReady(function () {
+                //// 这是k线图 展示的 7日均线和30日均线。
```
Comment (yukozh) [inline src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js]: Can you please remove the commented code?

File: src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js [hunk 2]
```diff
@@ -1,2 +1,96 @@
...
+            item.className = ''
+        })
+        dom.className = 'active'
+    }
+    var addInternalChangedEventListener = function () {
+        var _this = this;
+        var intervalDom = document.getElementById('interval')
+        // 切换产品周期
+        intervalDom.addEventListener('click', function (e) {
+            // e.target.dataset.value 这个就是获取的产品的周期
```
Comment (yukozh) [inline src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js]: Remove chinese comments

**Review by yukozh — CHANGES_REQUESTED** [review]

File: src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js (hunk 1 above)
Comment (qinxiaowen) [reply src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js]: fixed

File: src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js (hunk 2 above)
Comment (qinxiaowen) [reply src/Andoromeda.Kyubey.Portal/wwwroot/js/candlestick/index.js]: fixed

**Pull request merged**